docs/source/guides/installation_guide.md

```diff
@@ -26,7 +26,7 @@ or similar distros:
     - build-essential
         
 Here's a one-liner to install the above packages on linux:
-`sudo apt install python3-dev build-essential libffi-dev python3-pip`
+`sudo apt-get install python3-dev build-essential libffi-dev python3-pip`
```

Comment:
@SeanMikha - Does simply `apt` not work for you system (which platform is it)?

Reply:
I understand, `apt-get` is more universal... But I think, if one uses a system which doesn't have `apt`, they'd figure out